connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/CyH4TransportDriver.cpp

```diff
@@ -70,33 +68,24 @@ CyH4TransportDriver::CyH4TransportDriver(PinName tx, PinName rx, PinName cts, Pi
 
 {
     enabled_powersave = false;
-    bt_host_wake_active = false;
-
-    sleep_manager_lock_deep_sleep();        // locking deep sleep because this option 
-                                            // does not include a host wake pin
-    holding_deep_sleep_lock = true;
```

Comment:
please remove holding_deep_sleep_lock declaration from _class CyH4TransportDriver:public CordioHCITransportDriver_. we do not use it.